book/contributions/rec_using_inchi/recipes/rec_using_inchi.md

```diff
@@ -0,0 +1,107 @@
+    - How to identify a chemical behind an InChI or an InChIKey
+- Citation: 'Using InChI and InChIKey to annotate your chemical entities', The IUPAC FAIR Chemistry Cookbook, https://iupac.github.io/WFChemCookbook/recipes/rec_using_inchi.html
+- Reuse: This notebook is made available under a [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) license.
+```
+
+## Scenario
+
+So far, we are quite used to identifying chemical substances with a name, a drawing or a CAS Registry Number (CAS-RN). However, these identifiers have some issues that affect their usability for indexing and searching. A way to increase the FAIRness of our documents is to annotate our chemicals with InChI and InChiKeys, either in papers, data, or metadata files.
+
+Let's do it together!
+
+## Case 1: I can draw my chemical 
+
+Many molecular drawing packages like ChemDraw, <https://revvitysignals.com/products/research/chemdraw>, ChemSketch, <https://www.acdlabs.com/products/chemsketch/>, or MarvinSketch, <https://chemaxon.com/marvin>, can output the InChI and the InChIKey for a molecule drawn on them.
+
+![](../images/rec_using_inchi_fig1.png)
+
+Alternatively, we can use the CACTUS Chemical Identifier Resolver, <https://cactus.nci.nih.gov/chemical/structure> to get the InChI or the InChIKey of a drawn chemical.
+
+![](../images/rec_using_inchi_fig2.png)
+
+## Case 2: I can unambiguously name my chemical 
+
+If we can name our chemical (and this name is included in the chemical databases), you can use PubChem or ChemSpider searching features, or the CACTUS Chemical Identifier Resolver, <https://cactus.nci.nih.gov/chemical/structure>, to obtain the InChI or the InChIKey.
+
+![](../images/rec_using_inchi_fig3.png)
+
+If you can name the compound according to IUPAC systematic nomenclature, you can use the Open Parser for Systematic IUPAC nomenclature (OPSIN), to calculate (this is not a search) the InChI and the InChiKey. OPSIN can be downloaded (accessed on the command line) or used online at <https://opsin.ch.cam.ac.uk/index.html>.
+
+![](../images/rec_using_inchi_fig4.png)
+
+## Case 3: I have a MOL file for my chemical
+
+If you have a MOL or SDF file for our molecule, you can use the UniChem service, <https://www.ebi.ac.uk/unichem/> to upload the file and obtain its InChI and InChIKey.
+
+You can also use the following software that you install locally to process the MOL file:
+
+-   InChI core software, <https://www.inchi-trust.org/download-latest-inchi-standard-software/>.
+-   OpenBabel, <https://openbabel.org/wiki/Main_Page>,
+
+to convert a MOL file to InChI or InChIKey.
+
+Many common molecular drawing programs can also export and import MOL files. This way. they can also be used to convert a MOL file to an InChI or InChIKey. Make sure to check that isotope and stereochemistry information is preserved when importing and exporting MOL files.
+
+## Case 3: I have the SMILES string or the InChI for my chemical
```

Comment:
Should this be 3.1?

Reply:
No, I think this should be 4 and the later ones incremented by 1.